src/planscape/config/treatment_goals.json

```diff
@@ -0,0 +1,18 @@
+{
+    "treatment_goals": [
+        {
+            "category_name": "Fire Dynamics",
+            "questions": [
+                {
+                    "question_text": "Which areas within my planning area have the highest probability of high severity fire? ",
+                    "priorities": [
+                        "probabiity of high severity fire"
```

Comment:
this is a metric, correct?  You may need to represent the metric in terms of its entire hierarchical namespace, e.g. region-pillar-element-metric, and I recommend using the spaceless IDs (e.g. metric_name) rather than the display names which will have spaces.

Similarly, queries are going to be different from region to region, so I recommend adding that representation in here somewhere.

Reply:
Updated! Let me know if the new structure looks good